certified-connectors/dox42/apiDefinition.swagger.json

```diff
@@ -0,0 +1,81 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "dox42",
+    "description": "Easily make a dox42 service call",
```

Comment:
@dox42diegoschleis Can you please provide a detailed description of your connector which every one can understand.

Reply:
I have implemented the requested changes.